### PR TITLE
CTC-78

### PR DESCRIPTION
**Pull request by SimaoBolota-MetaCell**

### CODE REVIEW

File: napari_hub_cli/checklist/metadata.py [hunk 1]
```diff
@@ -248,10 +249,25 @@ def display_checklist(analysis_result):
...
+            section_title = feature.meta.section.title
+            section_statuses[section_title] |= feature.found == False
+    
+    last_section_status = dict(section_statuses)
+
     for feature in analysis_result.features:
         if feature.meta.section and feature.meta.section.title != previous_title:
-            console.print()
-            console.print(feature.meta.section.title, style="bold underline white")
+            if  last_section_status[feature.meta.section.title] == False:
```
Comment (aranega) [inline napari_hub_cli/checklist/metadata.py:265]: @SimaoBolota-MetaCell here, you can "invert" the two ifs and then simplify them from 
```python
if last_section_status[feature.meta.section.title] == True:
    ...
if last_section_status[feature.meta.section.title] == False:
    ...
```

to 

```python
if last_section_status[feature.meta.section.title]:
    ...
else:
    ...
```

**Review by aranega — CHANGES_REQUESTED** [review]
Thanks for the PR! Just a small adjustment and all will be perfect!

To report the modification in `extra`, can you just merge this branch to `extra-ctc78`? That's easier to keep to the two branches in sync.

File: napari_hub_cli/checklist/metadata.py (hunk 1 above)
Comment (SimaoBolota-MetaCell) [reply napari_hub_cli/checklist/metadata.py]: Thank you!

**Review by aranega — APPROVED** [review]
Thanks a lot @SimaoBolota-MetaCell :pray:

**Pull request merged**